File: Treinamento/Python/Exercícios/Exercícios_funções/Questão02.py
def verificacaoNumero(n):
    if n > 0:
        return 'P'
    else:
        return 'N'

def main():
    print('*** Positivo e negativo ***')
    n = int(input('Digite o número desejado: '))
    print(verificacaoNumero(n))

main()

File: Treinamento/C/Exercícios_funções/Questão02.c
#include <stdio.h>

char verificacaoNumero(int n) {
    if (n >= 0) {
        return 'P';
    } else {
        return 'N';
    }
}

int main() {
    int n;

    printf("Digite o número desejado: ");
    scanf("%i", &n);

    printf("%c\n", verificacaoNumero(n));
}
